platform/testing/end-to-end/cypress-best-practices-on-vsp.md

```diff
@@ -403,6 +403,72 @@ cy.axeCheck(); // Run the aXe check after expanding everything.
 
 Please note: Tests written with the form tester automatically check for accessibility, so this command does not need to be used explicitly in such tests.
 
+#### cy.axeCheck() configuration defaults
+
+The axe-core configuration object looks like this:
+
+```javascript
+let axeBuilder = {
+  runOnly: {
+    type: 'tag',
+    values: ['section508', 'wcag2a', 'wcag2aa'],
+  },
+  rules: {
+    'color-contrast': {
+      enabled: false,
+    },
+  },
+};
+```
+
+The [axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/cypress/support/commands/axeCheck.js) scans pages for Section 508, WCAG2.0 A, and WCAG2.0 AA rulesets every time it is called in a Cypress end-to-end test.
+
+The configuration object also disables color-contrast checks. [Disabling the color-contrast rule](https://www.deque.com/axe/core-documentation/api-documentation/#other-strategies) was carefully considered and discussed with the VA 508 office as a way to improve build times and reduce false positive tests. Teams should re-enable the color-contrast check on 1-2 tests per suite.
```

Comment:
Just a thought: Do you think we should draw more attention to _"Teams should re-enable the color-contrast check on 1-2 tests per suite."?_ This seems important.